test/setup/main.tf

```diff
@@ -16,13 +16,15 @@
 
 module "project" {
   source  = "terraform-google-modules/project-factory/google"
-  version = "~> 3.0"
+  version = "~> 7.0"
 
-  name              = "ci-cloud-storage"
-  random_project_id = "true"
-  org_id            = var.org_id
-  folder_id         = var.folder_id
-  billing_account   = var.billing_account
+  name                      = "ci-cloud-storage"
+  random_project_id         = "true"
+  org_id                    = var.org_id
+  folder_id                 = var.folder_id
+  billing_account           = var.billing_account
+  budget_amount             = var.budget_amount
```

Comment:
We definitely shouldn't need to set up a budget.

Reply:
I agree but it seems the project-factory module v7 requires it? https://registry.terraform.io/modules/terraform-google-modules/project-factory/google/7.0.0?tab=inputs

Reply:
It's optional. You might want to upgrade to the latest version (9.0.0): https://github.com/terraform-google-modules/terraform-google-project-factory

Reply:
let me try again with v6 which doesn't require it

Reply:
To be clear no version has ever required it.https://github.com/terraform-google-modules/terraform-google-project-factory/tree/v7.0.0#inputs

Reply:
Thanks, i was looking at the wrong thing